geographer/add_drop_rasters_mixin.py

```diff
@@ -1,7 +1,5 @@
 """Mixin that implements adding/dropping rasters."""
 
-from __future__ import annotations
```

Comment:
I think I would keep this import and not change the lines below... (In the other files, which imported `annotations` but didn't use it, I think it's good to remove the import.)